### ISSUE TITLE
EFS Endpoint support

### ISSUE THREAD

**Issue by solidaeon** [terraform-providers/terraform-provider-aws#12960]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Recent release of ECS Fargate to support mounting of EFS volumes.

### New or Affected Resource(s)

* aws_efs_file_system_endpoint

### Potential Terraform Configuration

```
resource "aws_efs_file_system" "foo" { ... }

resource "aws_efs_file_system_endpoint" "this" {
  file_system_id = aws_efs_file_system.foo.id
  posix_user = {
    user_id = var.posix_user_id
    group_id = var.posix_group_id 
  }
  owner = {
      user_id = var.owner_user_id
      group_id = var.owner_group_id 
  }
}
```

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000
* https://aws.amazon.com/jp/blogs/aws/amazon-ecs-supports-efs/

**Comment by DrFaust92** [comment]
duplicate of #12118 and addressed in #11965

**Comment by ewbankkit** [comment]
It looks like there were some additions to the ECS [`EFSVolumeConfiguration`](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_EFSVolumeConfiguration.html) structure (`volume.efs_volume` attribute of the [`aws_ecs_task_definition`](https://www.terraform.io/docs/providers/aws/r/ecs_task_definition.html) resource) in [AWS SDK v1.30.7](https://github.com/aws/aws-sdk-go/releases/tag/v1.30.7).

**Comment by solidaeon** [comment]
> duplicate of #12118 and addressed in #11965

thanks
